test/v3/models/downstream/BFLossResponseSpec.scala

```diff
@@ -135,9 +132,9 @@ class BFLossResponseSpec extends UnitSpec {
         HateoasWrapper(
           bfLossResponse,
           Seq(
-            Link(s"/individuals/losses/$nino/brought-forward-losses/lossId", GET, "self")
+            Link(s"/individuals/losses/$nino/brought-forward-losses/$lossId", GET, "self")
```

Comment:
just took a look at the hateoas links on the spec; looks like this should be retrieve (self), amend, and delete

Reply:
I was just doing the retrieve endpoint Hateoas, and this last test is for the amend endpoint.  (Was tempted to change it but it sounded like Naren was going to be doing the amend as part of his controller).